src/panel-battle.tsx

```diff
@@ -47,7 +49,7 @@ class BattlesPanel extends PSRoomPanel<BattlesRoom> {
 		return <div><a href={`/${battle.id}`} class="blocklink">
 			{minEloMessage && <small style="float:right">({minEloMessage})</small>}
 			<small>[{format}]</small><br />
-			<em class="p1">{battle.p1}</em> <small class="vs">vs.</small> <em class="p2">{battle.p2}</em>
+			<em class="p1">{battle.p1}</em> & <em class="p3">{battle.p3}</em> <small class="vs">vs.</small> <em class="p2">{battle.p2}</em> & <em class="p4">{battle.p4}</em>
```

Comment:
This still needs to support non-multi battles.

Reply:
You're probably better off reverting this part if you don't want to write the specialized code for it.

Reply:
Or this is a simple fix:

```ts
const p1 = battle.p1 + (battle.p3 ? " & " + battle.p3 : "");
const p2 = battle.p2 + (battle.p4 ? " & " + battle.p4 : "");
```

```suggestion
			<em class="p1">{p1}</em> <small class="vs">vs.</small> <em class="p2">{p2}</em>
```